app/src/ereferrals/res/values/donottranslate.xml

```diff
@@ -69,5 +69,8 @@
     <string name="category_option_group_set_uid" translatable="false">yVH67wmhryq</string>
     <string name="test_organisation_name">test_organisation</string>
     <string name="test_organisation_uid">9fV2JoHmO94</string>
+    <string name="GPS_PROVIDER_DEFAULT" translatable="false">default</string>
+    <string name="GPS_LATITUDE_DEFAULT" translatable="false">16.010790</string>
+    <string name="GPS_LONGITUDE_DEFAULT" translatable="false">96.852446</string>
```

Comment:
These are the coordinates for Myanmar. Maybe we can move this to some point in the lake Victoria, given that this app is mainly for Tanzania, Uganda and Mozambique